test/cli/repl/t16-prelude/expected

```diff
@@ -53,4 +53,20 @@ ok, loaded test/cli/repl/data.psv, 13 rows
 > - Core evaluation:
 [homer, 2]
 
+> > -- IsSome
+> - Core evaluation:
+[homer, [True],marge, [True]]
+
+> -- IsNone
+> - Core evaluation:
+[homer, [False],marge, [False]]
+
+> > --GetOrElse example
+>                     [34mλλλλ[0m
+REPL Error:
+Check error:
+  Unsolvable constraint at  1:19
+    1:19  [32mt[0m$[32m502[0m =: TemporalityJoin [32mt[0m$[32m0[0m Aggregate
+    1:19  [32mt[0m$[32m3[0m =: TemporalityJoin Aggregate [32mt[0m$[32m502[0m
```

Comment:
yeh - I'm not actually sure whether these leftover constraints should be an error, either